sci_bert/dataset_readers/ebmnlp.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict, List, Sequence, Iterable
+import os
+
+import itertools
+import logging
+
+from overrides import overrides
+
+from glob import glob
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.file_utils import cached_path
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.fields import MetadataField, TextField, SequenceLabelField, Field
+from allennlp.data.instance import Instance
+from allennlp.data.tokenizers import Token
+from allennlp.data.token_indexers import TokenIndexer, SingleIdTokenIndexer
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+def _is_divider(line: str) -> bool:
+    empty_line = line.strip() == ''
+    if empty_line:
+        return True
+    else:
+        first_token = line.split()[0]
+        if first_token == "-DOCSTART-":  # pylint: disable=simplifiable-if-statement
+            return True
+        else:
+            return False
+
+@DatasetReader.register("ebmnlp")
+class EBMNLPDatasetReader(DatasetReader):
```

Comment:
How similar or different this is from the Conll2003 dataset reader ?

Reply:
it's nearly identical except fewer params passed to datareader init.  there are options that shouldn't be available for PICO because of how data is annotated at `IO` only

Reply:
Does that mean we can reuse AllenNLP's dataset reader ?